modules/ROOT/pages/configure-alerting.adoc

```diff
@@ -43,11 +48,18 @@ spec:
 |===
 +
 . Run the following on the controller VM.
++
+----
+$ sudo gravity resource create -f alert-smtp.yaml
+----
+
+. Use the `rtfctl` utility to set the alert sender email address with your SMTP sending domain. We recommend using a distinguishable email address for each cluster to determine which cluster triggers the alert.
++
 ----
-$ gravity resource create -f alert-smtp.yaml
+$ sudo ./rtfctl appliance apply alert-smtp-from "sender@example.com"`
```

Comment:
Why does `rtfctl` require sudo?

Reply:
rtfctl needs to run as root to perform operations in the gravitational cluster.

Reply:
it should run with the same permission as `gravity` though - that needs to run with root permissions, so either we add `sudo` to gravity command or remove it from `rtfctl` command

Reply:
Got it. Yes, We should think about running out utils without escalated privilege (root) but let's continue since it depends on gravity which require `root`. Thanks for the info.